examples/python/trigger-pagerduty-incident/stack.yml

```diff
@@ -0,0 +1,15 @@
+provider:
+  name: openfaas
+  gateway: https://127.0.0.1:31112
+functions:
+  pdinvoke-fn:
+    lang: python3
+    handler: ./handler
+    image: pkbu/veba-pagerduty-invoke:latest
```

Comment:
Please replace with official VEBA VMware image

Reply:
do i then build and push to the vmware repository?

Reply:
I've updated the readme accordingly but i don't think i have permissions to push my docker build to the vmware repo. I'm leaving this pointing to my image at this time and have you guys update this for me once the image is made available from vmware repo. fair?

Reply:
For local testing, use your image. In this PR please replace this line with an official image from VMware. Once the PR is merged, we'll push the image to the VMware hub.

Reply:
done